client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/heuristic/TruthnessUtils.java

```diff
@@ -0,0 +1,32 @@
+package org.evomaster.client.java.instrumentation.heuristic;
+
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.DistanceHelper;
+
+public class TruthnessUtils {
+
+
+    public static Truthness getEqualityTruthness(long a, long b) {
+        double distance = DistanceHelper.getDistanceToEquality(a, b);
+        return new Truthness(
+                1d - distance,
```

Comment:
this assumes that a "distance" is always in [0,1].
I think we should relax such constraint.
The code was written with 2 different concepts: "heuristic" values in [0,1], and "distances" >=0.
In the end, the `core` module sees only `heuristics` values, and not "distances".   Normalization into [0,1] can lose precision, so it is best to do last when we need to compute the "heuristic" values

Reply:
I am refactoring this so DistanceHelper.getDistanceToEquality() returns a distance >=0 and then the distance is normalised to [0,1] (otherwise the truthness offset to true could be invalid). Is this correct?